plugins/religion.py

```diff
@@ -36,3 +36,381 @@ def koran(inp):  # Koran look-up plugin by Ghetto Wizard
         return 'No results for ' + inp
 
     return results[0].text_content()
+
+tao = ["""Tao called Tao is not Tao. Names can name no lasting name. 
```

Comment:
Change this to something other than `tao`, maybe `TAO`. Mostly because you're later redefining the name `tao` to mean your plugin function.